### ISSUE TITLE
Accessibility issue - Screen reader is not reading the "day" for the date selected in date picker

### ISSUE THREAD

**Issue by sivashree** [mezoistvan/ng2-flatpickr#122]
**Description**
Accessibility issue - When the page with ng2-flatpickr is used with a screen reader, **the screen reader doesn't announce the day for the date selected in date picker**. It just reads out the date, month and year.

**Expected behavior**
The screen reader should announce the day for the date selected.

**Actual behavior**
THe screen reader doesn't announce the day, it only announces the date, month and year for the date selected in date picker.

**Desktop (please complete the following information):**
 - Windows, MacOS
 - Chrome, firefox, edge
 - Screen readers - Voiceover in Safari, NVDA in Windows

**Additional context**
Please use any of the screen reader mentioned above and try to replicate the issue. Kindly reply with ETA for this fix so that our team can plan accordingly.
